<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>631</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMyRecipes">
     <attribute name="title">
      <string>My Recipes</string>
     </attribute>
     <widget class="QListWidget" name="recipeList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>41</y>
        <width>341</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAllRecipes">
     <attribute name="title">
      <string>All Recipes</string>
     </attribute>
     <widget class="QListWidget" name="recipeList_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>561</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Book Antiqua</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="PriceLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>41</y>
        <width>341</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
